### PR TITLE
lensfun: update to 0.3.3.

### PR DESCRIPTION
**Pull request by Vagabond**
#### Testing the changes
- I tested the changes in this PR: YES I tested by installing the new lensfun and checking the new lenses were available in darktable.

#### Local build testing
- I built this PR locally for my native architecture, (amd64)

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -1686,7 +1686,7 @@ libIlmImfUtil-2_4.so.24 libopenexr-2.4.0_1
 libGraphicsMagick.so.3 libgraphicsmagick-1.3.19_1
 libGraphicsMagick++.so.12 libgraphicsmagick-1.3.22_1
 libGraphicsMagickWand.so.2 libgraphicsmagick-1.3.19_1
-liblensfun.so.1 lensfun-0.3.2_1
+liblensfun.so.1 lensfun-0.3.3_1
```
Comment (tibequadorian) [inline common/shlibs:1689]: Don't do this if the soname doesn't change

Comment (Vagabond) [reply common/shlibs:1689]: How do you tell if it changed? It seems like it changed?

Comment (tibequadorian) [reply common/shlibs:1689]: The soname is in the first column and it didn't change. Second column is the package with the minimal version which provides this soname.

Comment (paper42) [reply common/shlibs:1689]: xbps-src would exit with an error if soname changed and you didn't change it in common/shlibs.

**Pull request closed without merging**